olive/systems/azureml/aml_system.py

```diff
@@ -405,6 +476,27 @@ def pass_runner_pipeline():
 
         return pass_runner_pipeline()
 
+    def _run_workflow_job(
+        self,
+        ml_client,
+        pipeline_job,
+        experiment_name: str,
+        tags: Dict = None,
+    ):
+        logger.debug("Submitting pipeline")
+        tags = {**self.tags, **(tags or {})}
+        job = retry_func(
+            ml_client.jobs.create_or_update,
+            [pipeline_job],
+            {"experiment_name": experiment_name, "tags": tags},
+            max_tries=self.azureml_client_config.max_operation_retries,
+            delay=self.azureml_client_config.operation_retry_interval,
+            exceptions=(HttpResponseError, JobException),
+        )
+        ml_client.jobs.stream(job.name)
```

Comment:
if we stream, the process keeps waiting for it to finish. I think we don't want that?

Reply:
After the job was submitted, it's okay to kill this streaming process, the job will be still running on the AML.

Reply:
Just wanted to make sure if the streaming and process kill is intended behavior. Or we just want to submit the job and return from the process without streaming.

Reply:
Keep this steaming is an easier way to track the workflow run status without having to check job status online, whether it succeed or fail. But user still can terminate this anytime, so I keep this streaming here.

Reply:
That makes sense. Maybe the log can tell the user it's safe to exit the program? So that they don't assume it needs to be kept alive

Reply:
Yeah i can add a log for this. I added such log before when running the workflow on a remote VM but not for AML computes. Azure cli already has such log if you terminate this: 
```
azure.ai.ml.exceptions.JobException: The output streaming for the run interrupted.
But the run is still executing on the compute target.
Details for canceling the run can be found here: https://aka.ms/aml-docs-cancel-run
```

Reply:
Since this log is printed only when terminated, I think an olive log saying it's safe to exit would be helpful.